src/components/token/TokenBalanceRow.js

```diff
@@ -6,41 +6,33 @@
  */
 
 import React from 'react';
-import { withRouter } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import { numberUtils } from '@hathor/wallet-lib';
-import { connect } from 'react-redux';
+import { useSelector } from 'react-redux';
 
-const mapStateToProps = state => ({
-  decimalPlaces: state.serverInfo.decimal_places,
-});
+function TokenBalanceRow({ tokenId, address, total, unlocked, locked }) {
+  const history = useHistory();
+
+  const decimalPlaces = useSelector(state => state.serverInfo.decimal_places);
 
-class TokenBalanceRow extends React.Component {
   /**
    * Redirects to token detail screen after clicking on a table row
    *
    * @param {String} uid UID of token clicked
    */
-  onRowClicked = () => {
-    this.props.history.push(`/address/${this.props.address}?token=${this.props.tokenId}`);
+  const onRowClicked = () => {
+    history.push(`/address/${address}?token=${tokenId}`);
   };
 
-  render() {
-    return (
-      <tr onClick={e => this.onRowClicked(this.props.address)}>
-        <td className="d-lg-table-cell pe-3">{this.props.address}</td>
-        <td className="d-lg-table-cell pe-3">
-          {numberUtils.prettyValue(this.props.total, this.props.decimalPlaces)}
-        </td>
-        <td className="d-lg-table-cell pe-3">
-          {numberUtils.prettyValue(this.props.unlocked, this.props.decimalPlaces)}
-        </td>
-        <td className="d-lg-table-cell pe-3">
-          {numberUtils.prettyValue(this.props.locked, this.props.decimalPlaces)}
-        </td>
-      </tr>
-    );
-  }
+  return (
+    <tr onClick={_e => onRowClicked(address)}>
```

Comment:
`address` is not used, can you refactor to 

```suggestion
    <tr onClick={onRowClicked}>
```

?

Reply:
You're right. Fixed on https://github.com/HathorNetwork/hathor-explorer/pull/316/commits/706be70540a39e6adc4c88aeff8a945ec611bbff